pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/RebalanceConfig.java

```diff
@@ -136,6 +136,12 @@ public class RebalanceConfig {
   @ApiModelProperty(example = "300000")
   private long _retryInitialDelayInMs = 300000L;
 
+  // Disk utilization threshold override. If set, this will override the default disk utilization threshold
+  // configured at the controller level. Value should be between 0.0 and 1.0 (e.g., 0.85 for 85%)
+  @JsonProperty("diskUtilizationThreshold")
+  @ApiModelProperty(example = "0.85")
+  private double _diskUtilizationThreshold = -1.0; // -1.0 means not set, use default
```

Comment:
nit: reword to "-1.0 means override is disabled, use controller config default"
Perhaps add this to the comments right above the property instead of here?